data/pid_data.py

```diff
@@ -2498,6 +2566,214 @@
+       'get_response': {'items': [{'name': 'fan_error_status', 'type': 'uint8',
+								  'labels': [(0, 'No Error'),
+											 (1, 'Main Fan Error'),
+											 (2, 'UPE Fans Error'),
+											 (3, 'Pinch Fan Error'),
+											 (4, 'Stack Fan Error'),
+											 (5, 'Stack Fan Error')],
+								  }]},
+       'get_sub_device_range': 0,
+       'name': 'FAN_ERROR_STATUS',
+       'set_request': {'items': [{'name': 'regulate_fan_speed', 'type': 'uint8',
+								 'labels': [(0, 'No Error'),
+											(1, 'Main Fan Error'),
+											(2, 'UPE Fans Error'),
+											(3, 'Pinch Fan Error'),
+											(4, 'Stack Fan Error'),
+											(5, 'Stack Fan Error')],
+								 }]},
+       'set_response': {'items': []},
+       'set_sub_device_range': 0,
+       'link': '',
+       'value': 33280},
+	  {'get_request': {'items': []},
+       'get_response': {'items': [{'name': 'max_lamp_power', 'type': 'uint8',
+								  'labels': [(1, '900 Watt'),
+											 (2, '1200 Watt'),
+											 (3, '1400 Watt')],
+								  }]},
+       'get_sub_device_range': 0,
+       'name': 'MAX_LAMP_POWER',
+       'set_request': {'items': [{'name': 'max_lamp_power', 'type': 'uint8',
+								 'labels': [(1, '900 Watt'),
+											(2, '1200 Watt'),
+											(3, '1400 Watt')],
+								 }]},
+       'set_response': {'items': []},
+       'set_sub_device_range': 0,
+       'link': '',
+       'value': 33792},
+			 
+	  {'get_request': {'items': []},
+       'get_response': {'items': [{'name': 'fade_on_comm_loss', 'type': 'uint8',
+								  'labels': [(1, 'Hold'),
+											 (2, '30 Sec'),
+											 (3, '60 Sec')],
+								  }]},
+       'get_sub_device_range': 0,
+       'name': 'LAMP_FADE_ON_COMM_LOSS',
+       'set_request': {'items': [{'name': 'fade_on_comm_loss', 'type': 'uint8',
+								 'labels': [(1, 'Hold'),
+											(2, '30 Sec'),
+											(3, '60 Sec')],
+								 }]},
+       'set_response': {'items': []},
+       'set_sub_device_range': 0,
+       'link': '',
+       'value': 33793},
+
+	  {'get_request': {'items': []},
+	   'get_response': {'items': [{'name': 'pan_lock', 'type': 'bool'}]},
+       'get_sub_device_range': 0,
+       'name': 'LOCK_PAN',
+       'set_request': {'items': [{'name': 'pan_lock', 'type': 'bool'}]},
+       'set_response': {'items': []},
+       'set_sub_device_range': 0,
+       'link': '',
+       'value': 34304},
+
+	  {'get_request': {'items': []},
+	   'get_response': {'items': [{'name': 'tilt_lock', 'type': 'bool'}]},
+       'get_sub_device_range': 0,
+       'name': 'LOCK_TILT',
+       'set_request': {'items': [{'name': 'tilt_lock', 'type': 'bool'}]},
+       'set_response': {'items': []},
+       'set_sub_device_range': 0,
+       'link': '',
+       'value': 34305},
+			 
+	  {'get_request': {'items': []},
+       'get_response': {'items': [{'name': 'free_motion', 'type': 'uint8',
+								  'labels': [(1, 'Normal Operation'),
+											 (2, 'Pan Tilt Free Motion'),
```

Comment:
Yes, we don't use tabs at all. Although FWIW I think the tab equivalent would be two spaces.

If/when you get a working editor config, you should add it here: https://github.com/OpenLightingProject/editor-configs